spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -112,6 +112,14 @@ def delete_tester_from_app(app_id: nil, tester_id: nil)
       handle_response(response)
     end
 
+    def search_for_tester_in_app(app_id: nil, text: nil)
+      assert_required_params(__method__, binding)
+      text = CGI.escape(text)
+      url = "providers/#{team_id}/apps/#{app_id}/testers?order=asc&search=#{text}&sort=status"
+      response = request(:get, url)
+      handle_response(response)
+    end
+
```

Comment:
Can we get a spec that `search_for_tester_in_app` calls the expected URL with params? ☺️ 

https://github.com/fastlane/fastlane/blob/master/spaceship/spec/test_flight/client_spec.rb#L169